src/cli.rs

```diff
@@ -141,6 +150,251 @@ pub(crate) fn parse_startup_args() -> Result<LdkUserInfo, ()> {
 	})
 }
 
+// Default datadir relative to home directory
+#[cfg(target_os = "windows")]
+const DEFAULT_BITCOIN_DATADIR: &str = "AppData/Bitcoin";
+#[cfg(target_os = "linux")]
+const DEFAULT_BITCOIN_DATADIR: &str = ".bitcoin";
+#[cfg(target_os = "macos")]
+const DEFAULT_BITCOIN_DATADIR: &str = "Library/Application Support/Bitcoin";
+
+// Environment variable/.env keys
+const BITCOIND_RPC_USER_KEY: &str = "RPC_USER";
+const BITCOIND_RPC_PASSWORD_KEY: &str = "RPC_PASSWORD";
+
+fn print_rpc_auth_help() {
+	let home_path = env::home_dir();
+	let home_path = match home_path {
+		Some(ref p) => p.to_str().unwrap_or("$HOME"),
+		None => "$HOME",
+	};
```

Comment:
So `env::home_dir()` is deprecated since 1.29 as it has some unexpected behaviour. The recommendation is to use something like [`dirs::home_dir`](https://doc.rust-lang.org/std/env/fn.home_dir.html). But that's another dependency (sigh), and in this case it seems to work (still gotta check windows). I'd say it's ok to keep using `env::home_dir()`.

Just a suggested DRYing up here:

```suggestion
	let home_path = env::home_dir().map(|ref p| p.to_str()).flatten().unwrap("$HOME");
```

Reply:
Oof I see my suggestion has `unwrap` instead of `unwrap_or`. Sorry for the typo!